app/src/hooks/usePpg.ts

```diff
@@ -0,0 +1,148 @@
+import axios from "axios";
+import React from "react";
+import { Alert } from "react-native";
+import OpenPGP from "react-native-fast-openpgp";
+import { UploadPGPKeyResponse } from "../types/general";
+import { PGP } from "../types/wallet";
+import { pgpLocalStorage } from "../utils/local-storage";
+import { createRandomPassword } from "../utils/randomPassword-utils";
+
+type Hooks = {
+  generateKeyPair: (
+    name: string | undefined,
+    email: string | undefined
+  ) => Promise<void>;
+  createPublicKey: (privateKey: string | undefined) => Promise<void>;
+  publishPGPPublicKey: (
+    publicKey: string | undefined,
+    email: string | undefined
+  ) => Promise<void>;
+  pgp: PGP | undefined;
+  setPGP: (pgp: PGP) => void;
+  verifyPGPPublicKey: (email: string | undefined) => Promise<void>;
+};
+
+const usePgp = (): Hooks => {
+  const [pgp, setPGP] = React.useState<PGP | undefined>(undefined);
```

Comment:
As far as I can tell this variable isn't used anywhere.  Is there a reason we have it?